submissions/Yasya23/holi-style-popup/style.css

```diff
@@ -0,0 +1,179 @@
+* {
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+:focus {
+  outline: none;
+}
+
+.navigation {
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  position: relative;
+  margin: 0.3rem 0.6rem 0.6rem 0.3rem;
+}
+
+.navigation__list {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: right;
+      -ms-flex-pack: right;
+          justify-content: right;
+  margin: 1rem;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+}
+
+.navigation__item {
+  margin-right: 0.6rem;
+  padding: 0.2rem;
+  max-width: 100%;
+  height: 100%;
+}
+
+.navigation__popup-icon {
+  cursor: pointer;
+}
+
+.popup {
+  position: relative;
+}
+
+.popup__input,
+.popup__more-input {
+  position: absolute;
+  left: 0.5rem;
+  top: 0.9rem;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
+}
+
+.popup__container {
+  display: none;
+  position: absolute;
+  top: 3rem;
+  right: -2.5rem;
+  width: 19rem;
+  max-height: 35rem;
+  -webkit-box-shadow: 0 1px 6px 0 rgba(32, 33, 36, 0.28);
+          box-shadow: 0 1px 6px 0 rgba(32, 33, 36, 0.28);
+  border-radius: 5px;
+  overflow: auto;
+}
+
+.popup__list {
+  display: grid;
+  grid-template: auto/1fr 1fr 1fr;
+  gap: 0.6rem;
+  padding: 1.2rem;
+}
+
+.popup__box-img {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+      -ms-flex-direction: column;
+          flex-direction: column;
+  -webkit-box-pack: end;
+      -ms-flex-pack: end;
+          justify-content: flex-end;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+  overflow: hidden;
+  padding: 0.3em 1rem;
+}
+
+.popup__box-img:hover,
+.navigation__item:hover {
+  border-radius: 5px;
+  background-color: rgb(217, 242, 255);
```

Comment:
here you are using RGB color model and below at the next class HEX model. it's not forbidden, but it is easier to mantain and to read when it is consistent, IMHO

Reply:
Changed, only the shadows remained in rgba